public/main.js

```
@@ -4,135 +4,134 @@
+document.getElementById("rollDices").addEventListener("click", function () {
+  connection.send(message("roll"));
+});
 
+connection.setOnmessage(function (event) {
+  var msgJson = JSON.parse(event.data);
+
+  console.debug(msgJson.type);
+
+  switch (msgJson.type) {
+    case "roll":
+      var diceLogEl = document.getElementById("diceLog");
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)